compiler/src/dotty/tools/dotc/typer/Applications.scala

```diff
@@ -185,6 +185,27 @@ object Applications {
       case _ => None
     }
   }
+
+  /** 1. If we are in an inline method but not in a nested quote, mark the inline method
```

Comment:
The code seems to mark a macro also for quotes. So it seems comment and code disagree. Which is right?

Reply:
The comment is right. We used to overapproximate and mark them all as macros which was not ideal. I added a commit fixing this.